src/scripts.js

```diff
@@ -114,10 +120,10 @@ function cardToggle(event) {
 function returnCardsToDefault(card) {
   switch (true) {
     case card === 'hydration':
-      cards[card].innerHTML = `<h3 class="hydration-station">Hydration Station</h3><p><strong>${dailyOunces} oz.</strong> of water consumed today</p><button class="hydration-station">More...</button>`
+      cards[card].innerHTML = `<h3 class="hydration-station">Hydration Station</h3><p>You drank <strong>${dailyOunces} oz.</strong> of water today</p><button class="hydration-station">More...</button>`
       break;
     case card === 'activity':
-      cards[card].innerHTML = `<h3 class="activity-tracker">Activity Tracker</h3><p>You have taken <strong>${dailySteps} steps</strong> today <p>[equivalent to ${dailyMiles} miles]</p></p><button class="activity-tracker">More...</button>` 
+      cards[card].innerHTML = `<h3 class="activity-tracker">Activity Tracker</h3><p>You have taken <strong>${dailySteps} steps</strong> today</p><button class="activity-tracker">More...</button>`
```

Comment:
I added it on the home page because it got wonky when I added text with the charts. I could just throw the miles (mentioned in the rubric) in one of the charts?